indico/modules/users/forms.py

```diff
@@ -75,6 +76,13 @@ class UserPreferencesForm(IndicoForm):
         widget=SwitchWidget(),
         description=_('Use Markdown editor instead of HTML editor when editing the minutes of a meeting.'))
 
+    mastodon_server_url = StringField(
+        _('Preferred Mastodon server'),
+        validators=[MastodonServer(), Optional()],
+        filters=[lambda x: strip_path_from_url(x) if x else x],
```

Comment:
Putting in an invalid URL swallows the error because `strip_path_from_url` returns an empty string in that case